WrightTools/artists/_helpers.py

```diff
@@ -946,32 +952,33 @@ def set_fig_labels(
     # get fig
     if fig is None:
         fig = plt.gcf()
+    # interpret row
+    numRows = fig.axes[0].numRows
+    if isinstance(row, int):
+        row %= numRows
+        row = slice(0, row)
+    row_start, row_stop, _ = row.indices(numRows)
+     # interpret col
+    numCols = fig.axes[0].numCols
+    if isinstance(col, int):
+        col %= numCols
+        col = slice(col, -1)
+    print(col)
+    col_start, col_stop, _ = col.indices(numCols)
+    print(row_start, row_stop, col_start, col_stop)
     # axes
     for ax in fig.axes:
         if ax.is_sideplot:
             continue
-        elif ax.is_first_col() and ax.is_last_row():
-            # lower left corner
-            set_ax_labels(
-                ax=ax,
-                xlabel=xlabel,
-                ylabel=ylabel,
-                xticks=xticks,
-                yticks=yticks,
-                label_fontsize=label_fontsize,
-            )
-        elif ax.is_first_col():
-            # lefthand column
-            set_ax_labels(
-                ax=ax, ylabel=ylabel, xticks=False, yticks=yticks, label_fontsize=label_fontsize
-            )
-        elif ax.is_last_row():
-            # bottom row
-            set_ax_labels(
-                ax=ax, xlabel=xlabel, xticks=xticks, yticks=False, label_fontsize=label_fontsize
-            )
-        else:
-            set_ax_labels(ax=ax, xticks=False, yticks=False)
+        if True: #row_start < ax.rowNum < row_stop and col_start < ax.colNum < col_stop:
```

Comment:
I think this is the correct set of truth statements. However, I don't think they are implemented fully below. For instance, a column after `col_stop` still has its ylabel changed. This problem also exists for rows before `row_stop` being changed. Consider the below example:
```
import numpy as np
import matplotlib.pyplot as plt
import WrightTools as wt

fig, gs = wt.artists.create_figure(nrows=2, cols=[1,1,'cbar'])

indxs = [(row, col) for row in range(2) for col in range(2)]

for i, indx in enumerate(indxs):
    ax = plt.subplot(gs[indx])
    ax.set_ylabel('y')
    ax.set_xlabel('x')
    
cax =   plt.subplot(gs[:,-1])  
cax.set_ylabel('cax')

row = slice(0, 1, None)
col = slice(0, 1, None)
wt.artists.set_fig_labels(row=row, col=col)
```
I expect the last column (`cax`) to not be touched w.r.t. its yaxis. It however is has its ticks and labels removed. 
![image](https://user-images.githubusercontent.com/15792657/48674344-a381d880-eb10-11e8-98ed-688f0c0498a6.png)

Reply:
try your code again---it should work this time